libraries/core-react/src/components/Search/Search.tsx

```diff
@@ -294,7 +267,6 @@ export const Search = forwardRef<HTMLInputElement, SearchProps>(function Search(
   const clearButtonProps = {
     isActive: state.isActive,
     size,
```

Comment:
Suggest removing `size` property as `Button` does not support it nor, our styled wrapper.